### PR TITLE
Added from_env_with_prefix and init benchmark system

### PR DESCRIPTION
**Pull request by wyatt-herkamp**
Node Operations for benchmarks can be added but this lays out the initial code.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
 mod cond_log;
 mod de;
-mod error;
+pub mod error;
 mod value;
 
-pub use de::from_env;
+pub use de::{from_env, from_env_with_prefix};
```
Comment (Xuanwo) [inline src/lib.rs:40]: Do you think it's a good idea to add a new API called `from_map` and impl `from_env` via `from_map` instead? In this way, users can implement features like `from_env_with_prefix` more easier.

Comment (wyatt-herkamp) [reply src/lib.rs:40]: What do you mean? Like allow developers to provide a map of environment variables?

Comment (wyatt-herkamp) [reply src/lib.rs:40]: It could be in addition to this. However, I think having the with_prefix is a nice to have.

Comment (wyatt-herkamp) [reply src/lib.rs:40]: I don't personally have a need to add from_map so maybe that should be done by someone else.

**Comment by wyatt-herkamp** [comment]
@Xuanwo anything blocking this from being merged?

**Pull request merged**